From 8c0c870cba8322a6896026cce83982e325a56351 Mon Sep 17 00:00:00 2001
From: hhdrake <hans.hageman@hotmail.com>
Date: Sun, 11 Sep 2022 11:44:24 +0200
Subject: [PATCH] printk i2c/ov5640.c

---
 drivers/media/i2c/ov5640.c | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/drivers/media/i2c/ov5640.c b/drivers/media/i2c/ov5640.c
index 1b2b947d7b3a..d56d9ba0cb20 100644
--- a/drivers/media/i2c/ov5640.c
+++ b/drivers/media/i2c/ov5640.c
@@ -3146,6 +3146,8 @@ static int ov5640_probe(struct i2c_client *client)
 	u32 rotation;
 	int ret;
 
+printk("HH - %s >>>>>>\n", __func__);
+
 	sensor = devm_kzalloc(dev, sizeof(*sensor), GFP_KERNEL);
 	if (!sensor)
 		return -ENOMEM;
@@ -3240,6 +3242,8 @@ static int ov5640_probe(struct i2c_client *client)
 
 	v4l2_i2c_subdev_init(&sensor->sd, client, &ov5640_subdev_ops);
 
+printk("HH - %s v4l2_i2c_subdev\n", __func__);
+	
 	sensor->sd.flags |= V4L2_SUBDEV_FL_HAS_EVENTS;
 	sensor->pad.flags = MEDIA_PAD_FL_SOURCE;
 	sensor->sd.entity.ops = &ov5640_sd_media_ops;
-- 
2.25.1
