From b574a9c0b2ceedd993ae2f389ec2175e6f609fb2 Mon Sep 17 00:00:00 2001
From: hhdrake <hans.hageman@hotmail.com>
Date: Fri, 9 Sep 2022 12:47:21 +0200
Subject: [PATCH] printk statement

---
 drivers/media/i2c/adv7180.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/drivers/media/i2c/adv7180.c b/drivers/media/i2c/adv7180.c
index 4498d14d3429..5cba881855db 100644
--- a/drivers/media/i2c/adv7180.c
+++ b/drivers/media/i2c/adv7180.c
@@ -1318,6 +1318,8 @@ static int adv7180_probe(struct i2c_client *client,
 	struct v4l2_subdev *sd;
 	int ret;
 
+printk("HH >>>>>>>>>>>>>>>>\n"); 
+
 	/* Check if the adapter supports the needed features */
 	if (!i2c_check_functionality(client->adapter, I2C_FUNC_SMBUS_BYTE_DATA))
 		return -EIO;
-- 
2.25.1

